tests/GetUniqueFirstLettersTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+class GetUniqueFirstLettersTest extends TestCase
+{
+    //protected $arrays;
+
+    protected function setUp(): void
+    {
+        //require '/scr/web/functions.php';
+    }
+
+    /**
+     * @dataProvider positiveDataProvider
+     */
+    public function testPositive($input, $expected)
+    {
+       //echo __DIR__ ;
+         require_once __DIR__ . '\..\src\web\functions.php';
```

Comment:
require should not be placed inside method

Reply:
It's a test from Hometask 6. Deleted.